engine/class_modules/sc_death_knight.cpp

```diff
@@ -5938,6 +6039,16 @@ struct obliterate_strike_t : public death_knight_melee_attack_t
     }
   }
 
+  int n_targets() const override
+  {
+    int num_targets = 0;
+    if ( p() -> covenant.deaths_due -> ok() )
+    {
+      num_targets = p() -> in_death_and_decay() ? deaths_due_cleave_targets : 0;
+    }
+    return num_targets;
+  }
```

Comment:
You could just replace this with (or even with 0 instead of calling death_knight_melee_attack_t::n_targets())
```suggestion
  int n_targets() const override
  {
      if ( p() -> covenant.deaths_due -> ok() && p() -> in_death_and_decay() )
        return deaths_due_cleave_targets;
      return death_knight_melee_attack_t::n_targets();
  }
```